eng/common/scripts/Submit-PullRequest.ps1

```diff
@@ -71,7 +71,9 @@ param(
 
   [boolean]$CloseAfterOpenForTesting=$false,
 
-  [boolean]$OpenAsDraft=$false
+  [boolean]$OpenAsDraft=$false,
+
+  [boolean]$AddBuildSummary=$true
```

Comment:
https://github.com/Azure/azure-sdk-tools/blob/main/eng/common/scripts/logging.ps1#L3C13-L4C2

We can default this similar to our logging.